worker/docker_client.py

```diff
@@ -228,8 +228,36 @@ def download_image(self, docker_image, loop_callback):
                     pass
                 loop_callback(status)
 
-    @wrap_exception('Unable to start Docker container')
-    def start_container(self, bundle_path, uuid, command, docker_image,
+    def create_cli_command(self, bundle_path, uuid, command, docker_image,
+                        request_network, dependencies, extra_args=[]):
+        bundle_stat = os.stat(bundle_path)
+        uid = bundle_stat.st_uid
+        gid = bundle_stat.st_gid
+
+        docker_bundle_path = '/' + uuid
+
+        # Set up the volumes.
+        volume_bindings = ['%s:%s' % (bundle_path, docker_bundle_path)]
+        for dependency_path, docker_dependency_path in dependencies:
+            volume_bindings.append('%s:%s:ro' % (
+                os.path.abspath(dependency_path),
+                docker_dependency_path))
+
+        # Create the container.
+        #command = 'bash -c; ' + '; '.join(self.docker_commands)
+        command = 'bash'
+
+        return 'docker run {} {} {} {} {} {} {}'.format(
+                ' '.join(extra_args),
+                ' '.join([ '-v {}'.format(v) for v in volume_bindings ]),
+                '-w="{}"'.format(docker_bundle_path),
+                '', #'--user={}:{}'.format(uid, gid),
+                '-e {}="{}"'.format('HOME', docker_bundle_path),
+                docker_image,
+                command
+        )
```

Comment:
We might have to be careful about quoting here (what happens if a dependency has a space in it?)